sql/updates/world/3.3.5/2017_10_20_99_world_alchemy_spec.sql

```diff
@@ -0,0 +1,111 @@
+SET @TEXTID := xxx; -- Need 1
+DELETE FROM `npc_text` WHERE `ID`=@TEXTID;
+INSERT INTO `npc_text` (`ID`, `text0_1`, `BroadcastTextID0`) VALUES
+(@TEXTID, "Forgetting your skill in Potion Mastery is not something to do lightly.$B$BAre you absolutely sure?", 21870);
+
+SET @MENUID := xxx; -- Need 3
+DELETE FROM `gossip_menu` WHERE `MenuID` BETWEEN @MENUID AND @MENUID+2;
+INSERT INTO `gossip_menu` (`MenuID`, `TextID`, `VerifiedBuild`) VALUES
+(@MENUID, 11076, 0), -- Transmutation
+(@MENUID+1, 11075, 0), -- Elixir
+(@MENUID+2, @TEXTID, 0); -- Potion
+
+DELETE FROM `gossip_menu_option` WHERE `MenuID` IN (8542, @MENUID, 8540, @MENUID+1, 7571, @MENUID+2);
+INSERT INTO `gossip_menu_option` (`MenuID`, `OptionID`, `OptionIcon`, `OptionText`, `OptionBroadcastTextID`, `OptionType`, `OptionNpcFlag`, `ActionMenuID`, `ActionPoiID`, `BoxCoded`, `BoxMoney`, `BoxText`, `BoxBroadcastTextID`, `VerifiedBuild`) VALUES
+(8542, 0, 0, "I wish to unlearn Transmutation Mastery.", 0, 1, 1, @MENUID, 0, 0, 0, "", 0, 0),
```

Comment:
This and the other two similar gossip options don't have a broadcast_id set because the female one contains a newline at the end, making the gossip option look a bit misaligned. The male one doesn't have this issue.

Reply:
```sql
INSERT INTO `broadcast_text` (`ID`, `MaleText`, `FemaleText`, `EmoteID1`, `EmoteID2`, `EmoteID3`, `EmoteDelay1`, `EmoteDelay2`, `EmoteDelay3`, `UnkEmoteID`, `Language`, `Type`, `SoundID1`, `SoundID2`, `PlayerConditionID`, `VerifiedBuild`) VALUES 
(21874, 'I wish to unlearn Potion Mastery.', 'I wish to unlearn Potion Mastery.\n', 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 24330),
(21875, 'I wish to unlearn Elixir Mastery.', 'I wish to unlearn Elixir Mastery.\n', 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 24330),
(21876, 'I wish to unlearn Transmutation Mastery.', 'I wish to unlearn Transmutation Mastery.\n\n', 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 24330);
```
^Legion broadcast_text entries (bruteforced)
Just checked them and even there is the newline, so it has to be blizzlike.
Another problem is in case you don't provide the broadcast_text id the locales are not available at all and you have to create custom locales which would be just redundant data.

tl;dr
insert the broadcast_text entries, even tho they contain newlines

Reply:
Makes sense, thanks :) done.